src/python/py_image_generation_pipelines.cpp

```diff
@@ -418,7 +465,19 @@ void init_image_generation_pipelines(py::module_& m) {
                 const py::kwargs& kwargs
             ) -> py::typing::Union<ov::Tensor> {
                 ov::AnyMap params = pyutils::kwargs_to_any_map(kwargs);
-                return py::cast(pipe.generate(prompt, image, mask_image, params));
+                ov::Tensor res;
+                {
```

Comment:
looks like we don't need extra scope with ` { } ` here as `else { } ` already creates required scope for `py::gil_scoped_release`

Reply:
Yes, removed.